xml/System.Security.Cryptography/RSACng.xml

```diff
@@ -335,13 +335,13 @@
         <returns>The encrypted data.</returns>
         <remarks>To be added.</remarks>
         <exception cref="T:System.ArgumentNullException">
-          <paramref name="data" /> is <see langword="null" />.  
+          <paramref name="data" /> is <see langword="null" />.
   
  -or-  
   
  <paramref name="padding" /> is <see langword="null" />.</exception>
         <exception cref="T:System.Security.Cryptography.CryptographicException">
-          <paramref name="padding" /> does not equal <see cref="P:System.Security.Cryptography.RSASignaturePadding.Pkcs1" /> or <see cref="P:System.Security.Cryptography.RSASignaturePadding.Pss" />.</exception>
+          <paramref name="padding" /> does not equal <see cref="P:System.Security.Cryptography.RSAEncryptionPadding.Pkcs1" />, or else the <see cref="P:System.Security.Cryptography.RSAEncryptionPadding.Mode" /> of <paramref name="padding" /> does not equal <see cref="F:System.Security.Cryptography.RSAEncryptionPaddingMode.Oaep" />.</exception>
```

Comment:
```suggestion
          <paramref name="padding" /> isn't equal to <see cref="P:System.Security.Cryptography.RSAEncryptionPadding.Pkcs1" />

-or-

<paramref name="padding" />.<see cref="P:System.Security.Cryptography.RSAEncryptionPadding.Mode" /> isn't  equal to <see cref="F:System.Security.Cryptography.RSAEncryptionPaddingMode.Oaep" />.</exception>
```

Reply:
I think that's the wrong "or", Maira.  The exception is thrown if the padding mode isn't (pkcs1 or oaep), the only two supported modes.  I read the "-or-" version as it throws always (throws if != pkcs1, and throws if != oaep... nothing is both, so throws always)

Reply:
I have copied this statement from the same documentation from the `Decrypt` method, and I think that one should work.

https://github.com/dotnet/dotnet-api-docs/blob/056a526ea788cabaad9370167040d514947715ca/xml/System.Security.Cryptography/RSACng.xml#L220-L230

Reply:
Ok, I'll reword my suggestion. If both conditions should be met, I think we should use **and** then. Perhaps we might also need to review the Decrypt exception condition @CXuesong.

Reply:
@bartonjs can you review my suggestion?

Reply:
`<paramref name="padding" />.<see cref="P:System.Security.Cryptography.RSAEncryptionPadding.Mode" /> isn't equal to <see cref="P:System.Security.Cryptography.RSAEncryptionPadding.Pkcs1" /> or <see cref="F:System.Security.Cryptography.RSAEncryptionPaddingMode.Oaep" />.</exception>` looks reasonable to me.

Except that P:System.Security.Cryptography.RSAEncryptionPadding.Pkcs1 should probably be F:System.Security.Cryptography.RSAEncryptionPaddingMode.Pkcs1